web/app/scripts/src/services/api.es.js

```diff
@@ -17,10 +22,16 @@ export const api = function ($http, PROJECT_SETTINGS) {
     }
 };
 
-export const service = [
-    '$http',
-    'PROJECT_SETTINGS',
-    api
-];
+export const module = {
+    moduleName: 'incuna.surveys-api',
+    componentName: 'API'
+};
+
+angular.module(module.moduleName, [ProjectConfig.moduleName])
+    .service([
+        '$http',
+        ProjectConfig.componentName,
+        API
```

Comment:
I really don't think it's worth it.

However, to get the best out of both, why not use [ng-annotate](https://github.com/olov/ng-annotate) so we can have minified js whilst using the magic injector syntax?
```js
var FooFunction = function (dep1, dep2) {};
module.service('Foo', FooFunction);
```